ote/src/cljs/ote/views/operator_users.cljs

```diff
@@ -70,6 +71,6 @@
      [:h2 name]
      (if loaded?
        [:div
-        [access-table e! access-users name]
+        [access-table e! access-users (get-in state [:params :operator-id])]
         [invite-member e! access-state (get-in state [:params :operator-id])]]
-       [spinner/primary-spinner])]))
+       [spinner/primary-spinner :id "waiting-loading-event-spinner"])]))
```

Comment:
I see a new spinner has been defined (by @jkorpelin) recently. 
However there already was a material-ui spinner `ote.ui.circular_progress` (which is css as well) which is used in a few places. 
There shouldn't be two overlapping implementations.
Shouldn't we use the material-ui component by default in order to avoid re-inventing the wheel?

Reply:
I made a ticket for this. Lets figure out what spinner we will use in the future.